packages/core/admin/admin/src/reducers.js

```diff
@@ -1,19 +1,21 @@
+import appReducer from './pages/App/reducer';
 import rbacProviderReducer from './components/RBACProvider/reducer';
-import appReducer from './content-manager/pages/App/reducer';
+import cmAppReducer from './content-manager/pages/App/reducer';
 import editViewLayoutManagerReducer from './content-manager/pages/EditViewLayoutManager/reducer';
 import listViewReducer from './content-manager/pages/ListView/reducer';
 import rbacManagerReducer from './content-manager/hooks/useSyncRbac/reducer';
 import editViewCrudReducer from './content-manager/sharedReducers/crudReducer/reducer';
 
 const contentManagerReducers = {
-  'content-manager_app': appReducer,
+  'content-manager_app': cmAppReducer,
   'content-manager_listView': listViewReducer,
   'content-manager_rbacManager': rbacManagerReducer,
   'content-manager_editViewLayoutManager': editViewLayoutManagerReducer,
   'content-manager_editViewCrudReducer': editViewCrudReducer,
 };
 
 const reducers = {
+  'strapi-admin_app': appReducer,
```

Comment:
Just a suggestion: no need to change it: What do you think about naming it `admin_app` instead? All the others aren't prefixed with strapi either and there isn't another admin ...